src/components/VColorPaletteMaterial.vue

```diff
@@ -219,6 +225,7 @@ export default {
 
   .text-field-hex {
     padding: 0px !important;
+    color:black !important;
```

Comment:
I've noticed few formatting issues, like this one. Add spacing between color and its value.